modules/vpc/variables.tf

```diff
@@ -0,0 +1,85 @@
+variable "tags" {
+  description = "A mapping of tags to assign to the resources"
+  type        = map(string)
+  default = {
+    terraform = true
+  }
+}
+
+variable "name" {
+  description = "The application name"
+  type        = string
+  default     = "nerves-hub"
```

Comment:
do we want a default here? ideally this should include the env or workspace name.

Reply:
Yeah probably best not to have a default here. Removed default.